Tests/js/views/ez-usermenuview.html

```diff
@@ -0,0 +1,58 @@
+<!doctype html>
+<html>
+<head>
+<title>eZ User Menu view tests</title>
+</head>
+<body>
+<div class="container"></div>
+
+<script type="text/x-handlebars-template" id="usermenuitemview-ez-template">
+    <a class="ez-user-menu-item">{{ title }}</a>
+</script>
+
+<script type="text/javascript" src="../../../Resources/public/vendors/yui3/build/yui/yui.js"></script>
+<script type="text/javascript" src="./assets/ez-usermenuview-tests.js"></script>
+<script>
+    var filter = (window.location.search.match(/[?&]filter=([^&]+)/) || [])[1] || 'raw',
+        loaderFilter;
+
+    if (filter == 'coverage'){
+        loaderFilter = {
+            searchExp : "/Resources/public/js/",
+            replaceStr: "/Tests/instrument/Resources/public/js/"
+        };
+    } else {
+        loaderFilter = filter;
+    }
+
+    YUI({
+        coverage: ['ez-usermenuview'],
+        filter: loaderFilter,
+        modules: {
+            "ez-usermenuview": {
+                requires: ['event-tap', 'event-outside', 'template-micro', 'ez-usermenuitemview'],
```

Comment:
`template-micro` and `ez-usermenuitemview` should not be there but `ez-view` should be added. That also means, `ez-usermenuitemview` and `ez-templatebasedview` modules can be removed below.